code/modules/xenoarcheaology/finds/find_types/mundane.dm

```diff
@@ -56,7 +56,7 @@
 /decl/archaeological_find/tank/spawn_item(atom/loc)
 	var/obj/item/tank/new_item = ..()
 	new_item.air_contents.gas.Cut()
-	new_item.air_contents.adjust_gas(pick(subtypesof(/decl/material/gas)),15)
+	new_item.air_contents.adjust_gas(pick(decls_repository.get_decls_of_subtype(/decl/material/gas)),15)
```

Comment:
I think this should also be `get_decl_paths_of_subtype()` since adjust_gas also takes a path?